### ISSUE TITLE
Request: Borderless window on Steam Big Picture Mode (windowed)

### ISSUE THREAD

**Issue by opethion** [Codeusa/Borderless-Gaming#233]
I would like to request the support for a Borderless window on Steam in Big Picture Mode (windowed).  I would just use the fullscreen option in Steam, but when I play a game that has a lower resolution than what my native monitor resolution is, upon exit of such a game, Steam shrinks up to the resolution of the game I was playing .  The only solution is to use Steam in Windowed mode, which retains window size upon exit, but I would love it if I could get the Borderless-Gaming program to make Steams Big Picture Mode (windowed) borderless.   Thank you.

**Comment by TrevorLeeC** [comment]
Yes! Borderless full screen is a need for steam big picture mode (Sbpm). Sbpm has so many issues running full screen that are annoying like the bug you mentioned. I run multiple monitors and having Sbpm running in full screen ruins the  Feng shui of my setup so I have to run in windowed version, but it looks so ugly in my opinion when you're forced to do that, and for me running Windows 10 operating system as well, whenever Sbpm starts in windowed version the title bar is hidden behind my dock and I have to reshape it then maximize the window. It's annoying these little bugs that could easily be fixed with a simple borderless window mode.
